Robust.Shared/GameObjects/Systems/SharedPointLightSystem.cs

```diff
@@ -87,4 +87,31 @@ public void SetSoftness(EntityUid uid, float value, SharedPointLightComponent? c
         comp.Softness = value;
         Dirty(uid, comp);
     }
+
+
+    /// <summary>
+    ///     Copy visuals from one pointlight to another.
+    /// </summary>
+    public void CopyVisuals(EntityUid uid, SharedPointLightComponent otherPointLight, SharedPointLightComponent? pointlight = null, bool forceDisabled = false)
```

Comment:
Use the modern `Entity<T>` way of passing in a paired uid and component.
```suggestion
    public void CopyVisuals(Entity<SharedPointLightComponent?> pointlight, SharedPointLightComponent otherPointLight, bool forceDisabled = false)
```

Reply:
Changed it, thanks for the hint! This is my first PR in both RobustToolbox and SpaceStation overall so I am glad for every suggestion.